src/components/RadioButton/RadioButton.js

```diff
@@ -27,7 +27,20 @@ import Icon from 'components/core/Icon';
 import { spacing, borderRadiusSizes } from 'utils/variables';
 import { isLightTheme } from 'utils/themes';
 
-const RadioButton = ({ visible }) => {
+// Constants
+import { OFFERS } from 'constants/exchangeConstants';
+
+const RadioButton = ({ visible, style, type }) => {
+  if (type === OFFERS) {
+    return (
+      <>
+        {visible && (
+          <RadioIcon style={style} name={isLightTheme() ? 'selected-radio-button' : 'radio-button-focus-icon'} />
+        )}
+        {!visible && <RadioIcon style={style} name={isLightTheme() ? 'radio-button' : 'radio-button-icon'} />}
+      </>
```

Comment:
So the radio button here isn't actually a radio button, its just two different icons, so it doesn't have any form element here. Is this how we want it?

Reply:
I followed the old structure for RadioButton. Just added a new radio button icon

Reply:
Yeah the naming here could change, since this isn't actually a button that can be pressed